VEDA_OS01/models.py

```diff
@@ -365,15 +365,14 @@ class Course (models.Model):
     )
 
     # TODO: Change field name
-    xue = models.BooleanField('Submit to VAL', default=False)
```

Comment:
I won't remove this in the same deployment, for safer side we've to remove the code first and then in next rollout we can remove fields.

Reply:
@muzaffaryousaf I was doing a "rename", (makemigration prompted for this, so it should be ok) since this field got repurposed. 
It's also, as far as I can tell, not being used anywhere in the codebase. I can definitely hold off if you think that's safer though.

Reply:
@muzaffaryousaf I should clarify - we have a few places where this is serialized, but it's not being used as a determiner for encodes by the pipeline.